### DISCUSSION TITLE
Push the 3.2.2 and 3.0.4 versions to the MVN repository.

### DISCUSSION THREAD

**Discussion by gaiyin** [apache/pulsar#22414] [Q&A] [answered]
I can't find 3.2.2 and 3.0.4 versions in the MVN repository (https://mvnrepository.com/).So can you push them pls. thanks.
@lhotari

**Comment by lhotari** [comment, marked as answer]
Unfortunately, there's some additional delay in the Maven central sync. That is currently being investigated.
I also notified the mailing list https://lists.apache.org/thread/4xlrlgky7dlbwsgz25rkq4nkhv569xvx . I contacted ASF Infra and they asked to create a Sonatype Central ticket. I have created the ticket and I'm waiting for the response.

In the meantime, it's possible to access the Maven artifacts directly from Apache Repository. The URL is https://repository.apache.org/content/repositories/releases . This can be configured in build tools such as Maven and Gradle.

**Reply by lhotari** [reply]
Update on the issue:
- 3.2.2 and 3.0.4 files are now available in Maven Central, but the maven-metadata.xml and directory listings aren't yet updated.

**Reply by lhotari** [reply]
The Maven central sync issue has been resolved. Thanks to Sonatype Maven Central support and ASF Infra team for resolving the issue.
